pkg/cmd/openshift-controller-manager/controller_manager.go

```diff
@@ -105,7 +105,7 @@ func RunOpenShiftControllerManager(config *openshiftcontrolplanev1.OpenShiftCont
 	return nil
 }
 
-func waitForHealthyAPIServer(client rest.Interface) error {
+func WaitForHealthyAPIServer(client rest.Interface) error {
```

Comment:
Please open an issue in library-go for us to add this function.

Reply:
If we do so, wouldn't it make sense to make the wait time configurable, through parameter?

Reply:
> If we do so, wouldn't it make sense to make the wait time configurable, through parameter?

yeah, we'd tweak a it a bit.  The issue is enough for us to get back to it later.

Reply:
https://github.com/openshift/library-go/issues/50